Rocket.Chat/Resources/en.lproj/Localizable.strings

```diff
@@ -299,6 +304,7 @@
 "myaccount.settings.profile.name_placeholder" = "Name";
 "myaccount.settings.profile.username_placeholder" = "Username";
 "myaccount.settings.profile.email_placeholder" = "E-mail";
+"myaccount.settings.profile.status.title" = "Status";
```

Comment:
@rafaelks `myaccount.settings.profile.status.title` is missing on the other languages files

Reply:
✅